wait-for-postgres.sh

```diff
@@ -17,10 +17,12 @@ if [[ $1 =~ $isport ]]; then
 fi
 cmd="$@"
 
-until PGPASSWORD=$POSTGRES_PASSWORD psql -h "$host" -p "$port" -U "postgres" -c '\q'; do
-  >&2 echo "Postgres is unavailable on $host:$port - sleeping"
-  sleep 1
-done
+if [ -z "$WAIT_FOR_POSTGRES" ]; then
```

Comment:
Would this not just test if WAIT_FOR_POSTGRES contains something (either true or false) - which would lead to confusion on the part of the user. Better would be an explicit test for "true", don't you think?
If i'm reading the code correctly, even _WAIT_FOR_POSTGRES: false_ would cause it to wait for postgres...